### PR TITLE
Removed unnecessary print statements

### PR DESCRIPTION
**Pull request by ZaiZu2**

### CODE REVIEW

**Review by Omastto1 — CHANGES_REQUESTED** [review]
add verbose argument instead of removing them

File: dbhydra/dbhydra_core.py [hunk 1]
```diff
@@ -1682,10 +1677,10 @@ def add_foreign_key(self, foreign_key):
         parent_id = foreign_key['parent_id']
         parent = foreign_key['parent']
         query = "ALTER TABLE " + self.name + " MODIFY " + parent_id + " INT UNSIGNED"
-        print(query)
+        pass
```
Comment (DovaX) [inline dbhydra/dbhydra_core.py:1680]: better full remove than replace for pass - or why is it done like that?

Comment (DovaX) [reply dbhydra/dbhydra_core.py:1680]: It's quite handy to see queries in the console btw - most of the time this is beneficial

**Review by DovaX — CHANGES_REQUESTED** [review]

**Comment by DovaX** [comment]
I am doing major refactor - some of the prints/logs are important, I close this PR for now

**Pull request closed without merging**